scripts/pulling_sim4life_images/pull_images.py

```diff
@@ -0,0 +1,180 @@
+import os
+import boto3
+import gitlab
+import git
+import shutil
+import sys
+from dotenv import load_dotenv
+import subprocess
+import yaml
+import paramiko
+
+
+
+# Load environment variables from the .env file
+load_dotenv()
+arg = sys.argv[1]
+
+
+
+stacks = {"master" : {"key": "osparc-infra/docs/deployments-hardware-and-credentials/osparc.speag.com/osparc-speag.pem", "e2e-name" : "Master", "sparc-external-name": "master", "registry" : "registry.osparc-master.speag.com"},
+"aws-staging" : {"key": "osparc-infra/docs/deployments-hardware-and-credentials/staging.osparc.io/osparc-staging.pem", "e2e-name" : "AWS-Staging", "sparc-external-name": "aws-staging", "registry" : "registry.staging.osparc.io" },
+"aws-production" : {"key": "osparc-infra/docs/deployments-hardware-and-credentials/osparc.io/osparc-production.pem", "e2e-name" : "AWS-Production", "sparc-external-name": "aws-production", "registry" : "registry.osparc.io" },
+"tip" : {"key": "osparc-infra/docs/deployments-hardware-and-credentials/tip.itis.swiss/osparc-public.pem", "e2e-name" : "tip.itis.swiss", "sparc-external-name": "tip-public", "registry" : "registry.tip.itis.swiss" },
+"osparc-staging.speag.com" : {"key": "osparc-infra/docs/deployments-hardware-and-credentials/osparc.speag.com/osparc-speag.pem", "e2e-name" : "Dalco", "sparc-external-name": "dalco-staging-production", "registry" : "registry.osparc-staging.speag.com" },
+"osparc.speag.com" : {"key": "osparc-infra/docs/deployments-hardware-and-credentials/osparc.speag.com/osparc-speag.pem", "e2e-name" : "Dalco", "sparc-external-name": "dalco-staging-production", "registry" : "registry.osparc.speag.com" },}
+
+args = ["master", "aws-production", "aws-staging", "dalco-staging", "dalco-production", "tip"]
+
+
+# This function parse yaml data from e2e-ops and return a list of hosts to ssh
```

Comment:
Optional Suggestion: Add to comment that this function expects e2e-ops to be pulled into `.`

Optional Enhancement: Pull the secondary git repos into a `.temp` folder